test/integration/etl_test.py

```diff
@@ -150,7 +150,7 @@ def test_extract_eia923(self, pudl_datastore_fixture):
         extractor = pudl.extract.eia923.Extractor(pudl_datastore_fixture)
         page = "stocks"
         year = 2018
-        if "Oil\r\nJune" not in extractor.load_source(page=page, year=year).columns:
+        if "Oil\nJune" not in extractor.load_source(page=page, year=year).columns:
```

Comment:
For some reason `calamine` 0.32 (which is wrapped by `python-calamine` 0.6) seems to have a different parsing behavior here -- it gets only a newline, not a carriage return + newline.  In practice I think this won't affect our parsing since we condense all whitespace and replace it with `_`.